staff/lbaella/JavaScript/calculation.js

```diff
@@ -0,0 +1,15 @@
+/*
+Calculation
+Write a program that writes in the console all the multiples of 23 less than 500 and at the end writes the sum of all of them
+*/
+
+function calculation() {
+    var sum = 0;
+    for (var i=0; i<=500; i+=23) {
+         console.log(i);
+         //sum = sum+i;
+         sum+=i;
+
+    }
+    console.log("SUMA "+sum)
```

Comment:
Let's start to get used to return values instead of `console.log` them
With this idea then you can `console.log` the values returned by the function easily